src/screens/wallet/wallet.tsx

```diff
@@ -27,37 +28,49 @@ export const Wallet = () => {
   const dispatch = useDispatch();
   const { navigate } = useNavigation();
 
+  const [isRefreshing, setRefreshing] = useState(false);
+
   const selectedAccount = useSelectedAccountSelector();
   const visibleAccounts = useVisibleAccountsListSelector();
   const tezosToken = useSelectedAccountTezosTokenSelector();
 
   usePageAnalytic(ScreensEnum.Wallet);
 
+  const handleRefresh = () => {
+    setRefreshing(true);
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 3000);
+  };
+
   return (
     <>
-      <HeaderCard hasInsetTop={true}>
-        <View style={WalletStyles.accountContainer}>
-          <CurrentAccountDropdown
-            value={selectedAccount}
-            list={visibleAccounts}
-            onValueChange={value => dispatch(setSelectedAccountAction(value?.publicKeyHash))}
-          />
+      {isRefreshing && <Divider />}
+      <ScrollView refreshControl={<RefreshControl refreshing={isRefreshing} onRefresh={handleRefresh} />}>
+        <HeaderCard hasInsetTop={true}>
+          <View style={WalletStyles.accountContainer}>
+            <CurrentAccountDropdown
+              value={selectedAccount}
+              list={visibleAccounts}
+              onValueChange={value => dispatch(setSelectedAccountAction(value?.publicKeyHash))}
+            />
 
-          <Divider />
+            <Divider />
 
-          <TouchableIcon name={IconNameEnum.QrScanner} onPress={() => navigate(ScreensEnum.ScanQrCode)} />
-        </View>
+            <TouchableIcon name={IconNameEnum.QrScanner} onPress={() => navigate(ScreensEnum.ScanQrCode)} />
+          </View>
 
-        <TokenEquityValue token={tezosToken} showTokenValue={false} />
+          <TokenEquityValue token={tezosToken} showTokenValue={false} />
 
-        <HeaderCardActionButtons token={tezosToken} />
+          <HeaderCardActionButtons token={tezosToken} />
 
-        <Divider size={formatSize(16)} />
+          <Divider size={formatSize(16)} />
 
-        <CollectiblesHomeSwipeButton />
-      </HeaderCard>
+          <CollectiblesHomeSwipeButton />
+        </HeaderCard>
 
-      <TokenList />
+        <TokenList />
```

Comment:
I think you have received error in console that `FlatList` in inside `ScrollView`